### DISCUSSION TITLE
anope_geoip_country vide - empty

### DISCUSSION THREAD

**Discussion by Ircgo** [anope/anope#429] [Support] [answered]
Bonjour,

Comme l'inique le titre, j'ai la table anope_geoip_country qui reste vide donc les informations ne sont pas transmise. J'utilise MagIRC et du coté Pays j'ai donc que inconnue.

**Comment by sadiepowell** [comment, marked as answer]
You need to run [geoipupdate.sh](https://github.com/anope/anope/blob/2.0/src/tools/geoipupdate.sh) to fill the geoip tables.
